doc/classes/ImmediateGeometry3D.xml

```diff
@@ -8,6 +8,7 @@
 		See also [ArrayMesh], [MeshDataTool] and [SurfaceTool] for procedural geometry generation.
 		[b]Note:[/b] ImmediateGeometry3D is best suited to small amounts of mesh data that change every frame. It will be slow when handling large amounts of mesh data. If mesh data doesn't change often, use [ArrayMesh], [MeshDataTool] or [SurfaceTool] instead.
 		[b]Note:[/b] Godot uses clockwise [url=https://learnopengl.com/Advanced-OpenGL/Face-culling]winding order[/url] for front faces of triangle primitive modes.
+		[b]Note:[/b] In case of missing points when handling large amounts of mesh data, try increasing its buffer size limit under [code]Project > Project Settings > Rendering > Limits > Buffers > Immediate Buffer Size Kb[/code].
```

Comment:
```suggestion
		[b]Note:[/b] In case of missing points when handling large amounts of mesh data, try increasing its buffer size limit under [member rendering/limits/buffers/immediate_buffer_size_kb].
```